csfix

```diff
@@ -0,0 +1,6 @@
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
hmm.... neviem ci robim nieco zle, ale narazil som na problem.
Kedze `csfix` pustam v dockeri (u seba nemam ani PHP nainstalovane) tak treba tento script presunut do `src` inak sa mi v dockeri neobjavi.

Reply:
Pravda, jasom pocital s externym php-cli, bude treba to presunut a zrejme potom spustat z dockera prikazom `docker-compose run php ./csfix`.